find.lua

```diff
@@ -64,93 +64,99 @@ local bwdDataAlgoNames = {
 
 local algoNames = {fwdAlgoNames, bwdFilterAlgoNames, bwdDataAlgoNames}
 
-local function call(layer, f, ...)
-   if find.verbose then
+local function getConvolutionDescriptor(desc)
+   local CUDNN_DIM_MAX=4
+
```

Comment:
just do 
```
local dimMax='CUDNN_DIM_MAX'
```
here, it will pick CUDNN_DIM_MAX value from ffi.lua. Besides, CUDNN_DIM_MAX=4 is not enough for volumetric convolutions?